[KOGITO-9311] Suggested improvements for the Kogito Serverless Operator Documentation (#364)
kiegroup/kogito-docs: serverlessworkflow/modules/ROOT/pages/cloud/operator/developing-workflows.adoc

:description: Developing workflow service with the operator on Kubernetes
:keywords: kogito, workflow, serverless, operator, kubernetes, minikube, devmode

This document describes how you can develop your Workflows directly on Kubernetes with the {operator_name}.

> Workflows in development profile are not tailored for production environments. To build and deploy an immutable Workflow application with the operator, see xref:cloud/operator/build-and-deploy-workflows.adoc[].

Reviewer: ```suggestion
Workflows in the development profile are not tailored for production environments. To build and deploy an immutable Workflow application with the operator, see xref:cloud/operator/build-and-deploy-workflows.adoc[].
```

    sw.kogito.kie.org/profile: dev <1>
spec: <2>
  flow:

>     specVersion: 0.8.0

Reviewer 2: This is a default value on the k8s api. Unfortunately some old versions of the server and kubectl won't replace it with the default value. We need to add a note here about this fix on newer kubectl versions. It's on my plate.

kubectl logs deployment/<workflow-name> -f
----
+
If you decide to open an issue or ask for help in {product_name} communication channels, this logging information is always useful for the person who will try to help you.

=== Possible Failure Scenarios

==== Feature Not Yet Supported 
The {operator_name} is under active development. Sometimes a feature might not be available yet. Please see xref:cloud/operator/known-issues.adoc[] for a comprehensive list of available features.

> If you identify you're attempting to use a feature not yet available, please file a new issue, so we can prioritize it for you. Alternatively, please ask in {product_name} communication channels.

Reviewer: ```suggestion
If you identify that you are attempting to use a feature that is not available yet, you can file a new issue, so we can prioritize it for you. Alternatively, you can ask in {product_name} communication channels.
```
